package.json

```diff
@@ -42,6 +42,7 @@
         "grunt-targethtml": "0.2.6",
         "grunt-template-jasmine-requirejs": "0.1.0",
         "grunt-usemin": "0.1.11",
+        "is-animated": "^1.1.1",
```

Comment:
if we're only using files we need from this in `thirdparty`, we don't need it to be included here.